MIP/mip-40/README.md

```diff
@@ -0,0 +1,83 @@
+
+Otherwise, the AB-FFS solution implemented implemented is referred to as *AB-FFS Decoupled* Grant. [MIP-n](todo) provides a more complete specification for *AB-FFS Decoupled* Grant.
+
+### L1 LP Token
+1. The L1 LP Token MUST NOT enshrine any special capabilities to mint or otherwise unlock the L1 staking token. That is, the L1 LP token must be independent of the L1 staking token and only effectively be used to purchase the L1 staking token.
+2. The governing system MAY provide vouchers or other means of redeeming the L1 LP token for the L1 staking token from a dedicate allocation. The object of these vouchers SHOULD be to peg the value of the L1 LP token to the L1 staking token.
+
+A more complete proposal for the usage L1 LP tokens with FFS is provided in [MIP-n](todo).
+
+### L2 Token Pool (Gas Pool)
+1. Burning gas an amount of gas $\alpha$ on the L2 MUST result in depositing that same amount of gas $\alpha$ into an L2 pool.
+2. The L2 pool MUST be recirculated to the L2 users in a manner that is not specified in this MIP. Generally, this means the L2 pool must either be used to issue rewards or otherwise be a reliable means of purchasing gas token on the L2. 
+
+A system adhering to these requirements is referred to as a Gas Pool. A more complete proposal for the usage of Gas Pools is provided in [MIP-n](todo).
+
+### Intermediary Tokens
+1. A complete implementation of *AB-FFS Decoupled* SHOULD use intermediary tokens to bridge the L1 and L2 systems. An implementation not using intermediary tokens is referred to as *AB-FFS Partially Decoupled*.
+2. The intermediary tokens MAY manage their own exchange rates, markets, and fees between each other, the L1 staking token, the L2 staking token, and the L1 LP token.
+
+A more complete proposal for the usage of intermediary tokens is provided in [MIP-n](todo).
+
+### Genesis
+[MD-38](https://github.com/movementlabsxyz/MIP/pull/38) requests provisions for genesis for any system addressing its desiderata. The genesis for *AB-FFS Decoupled* is not fully specified in this MIP and such full specification is provided under [MIP-n](todo). However, generally, there are two strategies for genesis:
+
+1. **Separate Mints**: mint the L1 staking token and the L2 gas token simultaneously in fixed supplies amounting to the intended total supply.
+2. **Genesis Bridge Transfer**: mint the L1 staking or L2 gas token in a fixed supply and transfer the other token from the L1 to the L2 s.t. a successful bridge transfer occurs before the system is operational.
+
+### On the Nature of the Tokens
+Fundamentally, the L1 staking and L2 gas token in an *AB-FFS Decoupled* system are independent, i.e., not constituting a representation of the same token redeemable on different chains. The comprise two closed token systems where assets cannot be lost or gained.
```

Comment:
The total supply in each remains fixed, which implies the supply across both remains fixed.

Reply:
ok i think i have a more fundamental issue here: the LP token has to have a value that is related to the cost of running `postconfirmation` (not FFS! since FFS-L2confirmation likely has to be rewarded by L2MOVE). 

`L1MOVE` has value since it is directly linked (1-1 swap - bridge - 1-1 swap) to `L2MOVE`, which is used for gas

but why would `LPStakingToken` have value?